### ISSUE TITLE
code within a link label does not look linkified

### ISSUE THREAD

**Issue by JakobJingleheimer** [nodejs/nodejs.org#7339]
### URL:

https://nodejs-pxw2fivbz-openjs.vercel.app/en/learn/modules/publishing-a-package

### Browser Name:

Brave (Chromium)

### Browser Version:

131.0.6778.108

### Operating System:

macOS 14.6

### How to reproduce the issue:

<img width="485" alt="code within a link has no styling of link" src="https://github.com/user-attachments/assets/67980ee3-9620-4f08-9c00-6e1d3f3a23ed" />

In the above, I would expect the `color` of `require()` to be the same as the link colour (green).

I believe the current is unintentional, and due to the style of `main code` having the same css specificity than `main a` (both `0,0,2`), but last wins, and `main code` occurs after.

I think `color` of link should have higher specificity to ensure it is not overridden.

**Comment by AugustinMauroy** [comment]
+1 to make `a` code as green

**Comment by mpruser** [comment]
Does this require input or agreement from multiple people? If it's okay, I can give it a try.

**Comment by AugustinMauroy** [comment]
Go ahead !

**Comment by ovflowd** [comment]
2cents: Instead of green make it have a green underline.

**Comment by JakobJingleheimer** [comment]
Hmm, I would keep it consistent: When two things are styled differently, I expect some kind of difference. If there's suddenly an underline that doesn't occur elsewhere, I would think "what is this underline" (whereas if the colour is the link colour, my brain already recognises that and will think "link").

**Comment by araujogui** [comment]
Fixed in #7386
